.circleci/config.yml

```diff
@@ -125,22 +125,6 @@ jobs:
           name: Clear Git changed files
           command: git checkout -- .
 
-  storybook_web_components:
-    <<: *image_defaults
-    steps:
-      - attach_workspace:
-          at: ~/
-      - run: yarn bootstrap
-      - run:
-          name: Build web components library
-          command: yarn build:wc
-      - run:
-          name: Deploy Storybook
-          command: yarn storybook:wc-deploy
-      - run:
-          name: Clear Git changed files
-          command: git checkout -- .
-
   static_analysis:
```

Comment:
deploying of web components storybook is now a git hub action and not a daily deploy removing legacy deploy